sdks/java/container/license_scripts/pull_licenses_java.py

```diff
@@ -140,17 +149,22 @@ def execute(dep):
     }
     '''
 
-    name = dep['moduleName'].split(':')[1].lower()
+    name = dep['moduleName'].split(':')[1]
     version = dep['moduleVersion']
     name_version = name + '-' + version
+    # javac is not a runtime dependency
+    if name == 'javac':
+      logging.debug('Skipping', name_version)
+      return
+    # skip self dependencies
+    if name.startswith('beam'):
```

Comment:
If I compare to the original code, it should be `dep['moduleName']` instead of `name`. If a beam package is 'beam:xxx' and the `xxx` does not has `beam` in it, it wouldn't skipped. And I also recommend applying `lower()` when checking if startswith 'beam'.

Reply:
Good catch. Changed it to be:

    if dep['moduleName'].lower().startswith('beam'):